src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.properties.Figure;
+import core.basesyntax.suppliers.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] drawings = new Figure[6];
```

Comment:
[All magic numbers in your code should be constants.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants)